### PR TITLE
Expire Password on Raspberry PI

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Addresses issue #198

### CODE REVIEW

File: scripts/install2 [hunk 1]
```diff
@@ -421,6 +425,12 @@ newhostname=$(sudo grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' |
 sudo echo "$newhostname" | sudo tee /etc/hostname > /dev/null
 sudo sed -i -e "s/$oldhostname/$newhostname/" /etc/hosts
 
+if [[ "$CHANGE_PASSWORD" == "pi" ]]; then
+    sudo passwd -e "$CHANGE_PASSWORD"
```
Comment (benhylau) [inline scripts/install2]: I didn't think sudo is required

File: scripts/install2 [hunk 2]
```diff
@@ -126,6 +129,7 @@ elif [[ "$BOARD_HARDWARE" == 'nanopineo2' || "$BOARD_HARDWARE" == 'sun50iw1p1' |
 else
     BOARD_FAMILY="Raspberry Pi"
     BOARD_REVISION="$(sed -rn 's/Revision\s+\:\s+([0-9a-z_\-\s\,\(\)]+)/\1/p' /proc/cpuinfo)"
+    CHANGE_PASSWORD="pi"
```
Comment (benhylau) [inline scripts/install2]: There is forcing password change when running the script. Perhaps the user already changed the password? Can we detect for default password?

File: scripts/install2 (hunk 1 above)
Comment (darkdrgn2k) [reply scripts/install2]: Seems only root can expire a password

```
pi@tomesh-6079:~ $  passwd -e pi
passwd: Permission denied.
pi@tomesh-6079:~ $ 
``

File: scripts/install2 (hunk 2 above)
Comment (darkdrgn2k) [reply scripts/install2]: Hard coded default password hash and check it against `/etc/shadow`

**Pull request merged**